### DISCUSSION TITLE
Way to reduse size of Songs?

### DISCUSSION THREAD

**Discussion by KoleckOLP** [ppy/osu#16485] [Ideas]
I have downloaded a bunch of song with video and I no longer want the video I would prefer to save up storage, I'd also like to get rid of song sounds, skins and such.

There is a tool for this for osu!stable https://github.com/henntix/osu-cleaner

The tool can't be used on osu!lazer because you no longer have access to song folders.

Probably the tool is not what eh devs would want you to use.

So don't we get a tool that does the same thing directly in lazer?

**Comment by peppy** [comment]
This is not supported in lazer. You can choose to download beatmaps without video at the point of downloading.

**Comment by KoleckOLP** [comment]
But what if I already have a bunch of songs with video and it's too many to go and redownload them all manually?

Shouln't there be a way?

**Reply by Gamingzone123** [reply]
if you delete the maps from lazer, run the cleanup on stable, then import beatmaps from stable, that should work

**Reply by KoleckOLP** [reply]
I mean that is how I did it but when lazer will be the only client in the future it won't be very practical or possible anymore.

**Reply by Gamingzone123** [reply]
true... lazer is constantly improving though
